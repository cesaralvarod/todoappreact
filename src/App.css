.container {
  max-width: 150rem;
  width: 95%;
  margin: 0 auto;
}
