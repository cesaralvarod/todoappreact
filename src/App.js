import React from "react";

// Components

import TopBar from "./components/layouts/TopBar";
import Content from "./components/Content";
import Footer from "./components/layouts/Footer";

// Styles

import "./App.css";

const App = () => {
  return (
    <>
      <TopBar />
      <div className="container">
        <Content />
      </div>
      <Footer />
    </>
  );
};

export default App;
