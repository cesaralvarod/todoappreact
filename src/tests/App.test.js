import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("Testing in component App", () => {
  test("Component App should show correctly", () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
