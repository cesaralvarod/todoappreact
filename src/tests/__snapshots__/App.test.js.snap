// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in component App Component App should show correctly 1`] = `
<Fragment>
  <TopBar />
  <div
    className="container"
  >
    <Content />
  </div>
  <Footer />
</Fragment>
`;
